qiskit/backends/ibmq/ibmqbackend.py

```diff
@@ -139,3 +141,59 @@ def status(self):
             raise LookupError(
                 "Couldn't get backend status: {0}".format(ex))
         return status
+
+    def jobs(self, limit=50, skip=0, status=None):
+        """Attempt to get the jobs submitted to the backend
+
+        Args:
+            limit (int): number of jobs to retrieve
+            skip (int): starting index of retrieval
+            status (None or JobStatus or str): only get jobs with this status,
+                where status is e.g. JobStatus.RUNNING or 'RUNNING'.
+        Returns:
+            list(IBMQJob): list of IBMQJob instances
+        """
+        backend_name = self.configuration['name']
+        job_list = []
+        base_index = skip
+        job_info_list = self._api.get_jobs(limit=limit, skip=base_index,
```

Comment:
a bit of code restructuring:
I think the 2 calls to the API here can be made just one. Place it in the first line of the `while` loop. Then if all the jobs are retrieved in the first call, the while loop terminates. The way it is written right now, at least 2 calls are made every time.